src/raw_field.rs

```diff
@@ -64,6 +64,18 @@ macro_rules! _memoffset__field_check {
 }
 
 /// Deref-coercion protection macro.
+#[cfg(allow_clippy)]
+#[macro_export]
+#[doc(hidden)]
+macro_rules! _memoffset__field_check_tuple {
+    ($type:ty, $field:tt) => {
+        // Make sure the type argument is a tuple
+        #[allow(clippy::unneeded_wildcard_pattern)]
+        let (_, ..): $type;
+    };
+}
+
+#[cfg(not(allow_clippy))]
```

Comment:
For consistency with the other "2 alternative version" macros, please don't leave an empty line here.

Reply:
Good catch! Fixed in 5ce2be6125ce7efc3b7bf735bdba98babb003de3 (force-pushed).